toolkit/scss/_temporary-message.scss

```diff
@@ -9,6 +9,7 @@
   background: $govuk-blue;
   color: $white;
   padding: 15px 15px 18px 15px;
+  margin-bottom: 20px;
```

Comment:
I think he means do `margin-bottom: $gutter;` where `$gutter == 30px`.  

Not sure when we use paddings and when we use margins though, if that's your question.